code/game/objects/structures/aliens.dm

```diff
@@ -279,7 +279,7 @@
 	//destroy any non-node weeds on turf
 	var/obj/structure/alien/weeds/check_weed = locate(/obj/structure/alien/weeds) in loc
 	if(check_weed && check_weed != src)
-		qdel(check_weed)
+		QDEL_IN(check_weed, 0) // Weeds might not be finished initializing yet (as in create_and_destroy).
```

Comment:
QDEL_IN(0) reminds me of sleep(0). Leaves a bad taste.

How about moving all this code
https://github.com/tgstation/tgstation/blob/12b82e9837d9bec50d3a0d6cfd180f36ca51a705/code/game/objects/structures/aliens.dm#L280-L282

To `LateInitialize()`? That should get called after all the weed and everything else in the game has initialized?

You should also start processing there.